source/Camera/InfoUtilOther/WarmupSample/WarmupSample.cs

```diff
@@ -0,0 +1,82 @@
+﻿
+/*
+This example shows a basic warm up method for a Zivid camera.
+*/
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.Globalization;
+using System.Threading;
+using Duration = Zivid.NET.Duration;
+
+
+
+namespace WarmupSample
+{
+	class Program
+	{
+		static void Main(string[] args)
+		{
+			try
+			{
+
+				var warmupTime = TimeSpan.FromMinutes(10);
+				var captureCycle = TimeSpan.FromSeconds(5);
+				var maxCaptureTime = Duration.FromMilliseconds(1000);
+
+				var zivid = new Zivid.NET.Application();
+
+				Console.WriteLine("Connecting to camera");
+				var camera = zivid.ConnectCamera();
+
+				Console.WriteLine("Getting suggested parameters");
+
+				var suggestSettingsParameters = new Zivid.NET.CaptureAssistant.SuggestSettingsParameters{
+					AmbientLightFrequency =
+					Zivid.NET.CaptureAssistant.SuggestSettingsParameters.AmbientLightFrequencyOption.none,
+					MaxCaptureTime = maxCaptureTime
+				};
+				var settings = Zivid.NET.CaptureAssistant.Assistant.SuggestSettings(camera, suggestSettingsParameters);
+
+				DateTime beforeWarmup = DateTime.Now;
+
+				Console.WriteLine("Starting warm up");
```

Comment:
In cpp we're also writing how long it will run.

        std::cout << "Starting warm up for: " << warmupTime.count() << " minutes" << std::endl;
        
https://github.com/zivid/zivid-cpp-samples/blob/2376e4186c835af2430f7f320e2702ef2ba8c9e5/source/Camera/InfoUtilOther/Warmup/Warmup.cpp#L34